File: JavaScript/Arrays&Hashing/TwoSum.js
 function twoSum(nums, target) {
        for (let i = 0; i < nums.length; i++) {
            for (let j = i + 1; j < nums.length; j++) {
                if (nums[i] + nums[j] === target) {
                    return [i, j]; 
                }
            }
        }
        return [];
};

let nums = [4,5,6];
let target = 10

console.log(twoSum(nums,target))

File: Python/Arrays&Hassing/TwoSum.py
# Number should be equal to input number
#  0  1  2  3
# [3][4][5][6]

# 0 -> 3
# 1 -> 4
# 2 -> 5
# 3 -> 6

# 3 + 4 = 7 
# 0 +1  = 7

# Brute Fore 
def twoSum(nums, target):
    for index in range(len(nums)):
        for j in range(index+1, len(nums)):
            if nums[index] + nums[j] == target:
                return [index,j]
            else:
                return []


nums = [3,4,5,6]
target = 7

twoSum(nums,target)
